src/seedu/addressbook/Main.java

```diff
@@ -88,8 +88,16 @@ private void runCommandLoopUntilExitCommand() {
             recordResult(result);
             ui.showResultToUser(result);
 
+            if(userCommandText.contains("add") || userCommandText.contains("delete")) {
+                command = new Parser().parseCommand("list");
+                CommandResult result1 = executeCommand(command);
```

Comment:
Not a good idea to name your variables like result1, result2 etc. You can try to give a more informative name